### DISCUSSION TITLE
Github修改代码的统计占比 | Sparkle的个人博客

### DISCUSSION THREAD

**Discussion by giscus** [iamspark1e/iamspark1e#1] [General]
# Github修改代码的统计占比 | Sparkle的个人博客

Github修改代码的统计占比, 前往我的博客了解更多。

https://sparkle.im/post/github%E4%BF%AE%E6%94%B9%E4%BB%A3%E7%A0%81%E7%9A%84%E7%BB%9F%E8%AE%A1%E5%8D%A0%E6%AF%94

**Comment by iWorld-y** [comment]
确实有用， 感谢
